### PR TITLE
Feature/increase visibility of last add button on document

### PR DESCRIPTION
**Pull request by LarsTheGlidingSquirrel**
Before: Plus buttons only appear on focused elements or on hover.
Changes: Last plus button in document editor is always visible and shows localized string ("Add an element") next to it. 

Styling will be addressed as a followup. 

Closes https://github.com/serlo/serlo-editor-issues-and-documentation/issues/96

### CODE REVIEW

File: packages/plugins/rows/src/index.tsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ export function createRowsPlugin(
 ): EditorPlugin<RowsPluginState, RowsConfig> {
   const { content } = config
 
+  console.log('Banana') // @@@
```
Comment (elbotho) [inline packages/plugins/rows/src/index.tsx:25]: 🍌 😂

Comment (elbotho) [reply packages/plugins/rows/src/index.tsx:25]: love it, way better than the test strings I usually use.

Comment (LarsTheGlidingSquirrel) [reply packages/plugins/rows/src/index.tsx]: Damn! Just tried to remove it but you were too quick! :laughing: 
Banana is just a nice word to type.

File: packages/plugins/rows/src/editor/separator.tsx [hunk 2]
```diff
@@ -79,16 +90,23 @@ export function Separator({
...
+  visuallyEmphasizedAddButton = false,
 }: {
   config: RowsPluginConfig
   isFirst?: boolean
   onClick: () => void
   focused?: boolean
+  visuallyEmphasizedAddButton?: boolean
 }) {
   return (
     <StyledSeparator isFirst={isFirst}>
```
Comment (elbotho) [inline packages/plugins/rows/src/editor/separator.tsx:102]: Even though the styling is all right for now, I'd love some top-margin.
Cleanest way I see is to give the last `StyledSeparator` `positition: static` (or not set `absolute`) with something like:

``` jsx
 <StyledSeparator isFirst={isFirst} isLast={isLast} >
 ```

**Review by elbotho — COMMENTED** [review]
looks good, added two suggestions.

File: packages/plugins/rows/src/editor/separator.tsx (hunk 2 above)
Comment (elbotho) [reply packages/plugins/rows/src/editor/separator.tsx:102]: Something like that would work nicely

image

``` css
{
    position: static;
    margin-bottom: 5rem;
    margin-top: -3rem;
}
```

Comment (LarsTheGlidingSquirrel) [reply packages/plugins/rows/src/editor/separator.tsx]: Thanks for looking into it. I already implemented it in a different way however.

**Review by elbotho — APPROVED** [review]
👍

**Pull request merged**